catalyst_voices/packages/internal/catalyst_voices_models/lib/src/optional.dart

```diff
@@ -23,7 +23,7 @@ import 'package:equatable/equatable.dart';
 ///   }
 /// }
 ///```
-final class Optional<T extends Object> extends Equatable {
+final class Optional<T extends Object?> extends Equatable {
```

Comment:
whats the reason for this change? Feels of to me

Reply:
Previous notation doesn't allow `Object?` to be used inside `Optional`.

Reply:
if you needed a null `Optional.empty` should be used or `Optional.of`. data field inside `Optional` is nullabe and this `T` parameter should not be nullable. 

Please double check your use case for this change

Reply:
I'll recheck, what you wrote makes sense

Reply:
Ok, my bad, I'm reverting this. At that time remember the compiler complaining about it but now I don't have any issue :/

Reply:
Fixed